models/database.go

```diff
@@ -111,6 +111,8 @@ var databaseSchema = [][]string{
 			username VARCHAR CHECK (username <> ''),
 			password VARCHAR CHECK (password <> ''),
 			metrics_url VARCHAR CHECK (metrics_url <> ''),
+			tls BOOLEAN,
+			tls_skip_verify BOOLEAN NULL,
```

Comment:
It should be a pointer in model, or non-NULLable column. In reform, `nil == NULL`.